af/fastpath/fastpath/core.py

```diff
@@ -59,7 +58,28 @@
 metrics = setup_metrics(name="fastpath")
 
 conf = Namespace()
-fingerprints: Dict[str, Dict[str, list]]
+fingerprints_update_time = 0
+
+FINGERPRINT_SCOPE_TO_LOCALITY = {
+    "inst": "local",
+    "isp": "isp",
+    "nat": "country",
+    "prod": "general",
+}
+
+
+class Fingerprint(TypedDict):
+    name: str
+    scope: str
+    other_names: str
+    location_found: str
+    pattern_type: str
+    pattern: str
+    confidence_no_fp: int
+    expected_countries: str
```

Comment:
I think the metadata we only really care to have in the scores dict is:
* name
* scope
* location_found
* confidence_no_fp
* expected_countries

The others are probably OK to exclude

Reply:
Added.